### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by momo-i**
Add Japanese translation

### CODE REVIEW

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]
Thank you for the contribution! Just a couple minor things, and this is good to go! And if you'd like to continually update the translation, feel free to join [our Discord server](https://discord.gg/U8DvBAW), and we'll assign you the Translators role. That will give you access to a channel where we post new lines that need translating a couple days before an update to the mod, so the translations can stay up to date.

File: src/main/resources/assets/securitycraft/lang/ja_jp.json [hunk 1]
```diff
@@ -0,0 +1,1196 @@
...
+
+	"gui.securitycraft:scManual.disabled": "無効",
+	"gui.securitycraft:scManual.recipe.keycard_lv1": "無効",
+	"gui.securitycraft:scManual.recipe.keycard_lv2": "無効",
+	"gui.securitycraft:scManual.recipe.keycard_lv3": "無効",
+	"gui.securitycraft:scManual.recipe.keycard_lv4": "無効",
+	"gui.securitycraft:scManual.recipe.keycard_lv5": "無効",
+	"gui.securitycraft:scManual.recipe.limited_use_keycard": "無効",
+	"gui.securitycraft:scManual.intro.1": "SecurityCraftを始めるにあたって",
+	"gui.securitycraft:scManual.intro.2": "by Geforce, bl4ckscor3, Redstone_Dubstep and ChainmailPickaxe",
```
[the remark this answers is not in the document]
Comment (momo-i) [reply src/main/resources/assets/securitycraft/lang/ja_jp.json:688]: Added author to line (translated to Japanese)

File: src/main/resources/assets/securitycraft/lang/ja_jp.json [hunk 2]
```diff
@@ -0,0 +1,1196 @@
...
+	"messages.securitycraft:ownable.ownerNotValidated": "このブロックの所有者はユニバーサルオーナーチェンジャーによって変更されたため、新しい所有者が再び機能することを確認する必要があります。ブロックを検証するには、ブロックの新しい所有者がブロックを右クリックする必要があります。",
+	"messages.securitycraft:sonic_security_system.notLinked": "まだブロックにリンクされていません。 最初にこのアイテムを手に持っている互換性のある SecurityCraft ブロックを右クリックしてください。",
+	"messages.securitycraft:sonic_security_system.blockLinked": "%2$sで%1$sとリンクしました。",
+	"messages.securitycraft:sonic_security_system.blockUnlinked": "%2$sで%1$sとのリンクが解除されました。",
+	"messages.securitycraft:sonic_security_system.linkMaxReached": "追加のブロックにリンクすることはできません。 各音波セキュリティシステムは、最大%sブロックにリンクできます。",
+	"messages.securitycraft:sonic_security_system.locked": "この%sは音波セキュリティシステムによってロックされています！",
+	"messages.securitycraft:portable_tune_player.tune_saved": "音楽を保存しました。",
+	"messages.securitycraft:portable_tune_player.no_tune": "利用可能な音楽はありません。",
+	"messages.securitycraft:sc_help": "%s: %s + %s",
+	"messages.securitycraft:formattedCoordinates": "X: %1$s Y: %2$s Z: %3$s",
```
[the remark this answers is not in the document]
Comment (momo-i) [reply src/main/resources/assets/securitycraft/lang/ja_jp.json]: Two lines removed.

File: src/main/resources/assets/securitycraft/lang/ja_jp.json [hunk 3]
```diff
@@ -0,0 +1,1196 @@
...
+	"messages.securitycraft:sentry.descriptionMode4": "(常に有効でプレイヤーに対して攻撃)",
+	"messages.securitycraft:sentry.descriptionMode5": "(プレイヤーが近くに来た時に有効になり、プレイヤーに対して攻撃)",
+	"messages.securitycraft:sentry.descriptionMode6": "(有効ではないので、何に対しても攻撃しない)",
+	"messages.securitycraft:blockpocket.multipleManagers": "構造内に複数のポケットブロックマネージャーがあります。",
+	"messages.securitycraft:blockpocket.alreadyAssembled": "ポケットブロックは組み立て済みです！",
+	"messages.securitycraft:blockpocket.invalidBlock.rotation": "ポケットブロックマネージャーの柱%1$sの向きが正しくありません！",
+	"messages.securitycraft:blockpocket.invalidBlock": "無効なブロック\"%2$s\"ポケットブロックマネージャ%1$s! (予想: \"%3$s\")",
+	"messages.securitycraft:blockpocket.blockInWay": "\"%2$s\"はポケットブロックマネージャーのスペース%1$sをブロックしています！",
+	"messages.securitycraft:blockpocket.unowned": "ポケットブロックマネージャーのブロック\"%2$s\"%1$sを所有していません！",
+	"messages.securitycraft:blockpocket.position.1": "%s",
```
[the remark this answers is not in the document]
Comment (momo-i) [reply src/main/resources/assets/securitycraft/lang/ja_jp.json]: position.1 line removed.

File: src/main/resources/assets/securitycraft/lang/ja_jp.json [hunk 4]
```diff
@@ -0,0 +1,1196 @@
...
+	"messages.securitycraft:sonic_security_system.locked": "この%sは音波セキュリティシステムによってロックされています！",
+	"messages.securitycraft:portable_tune_player.tune_saved": "音楽を保存しました。",
+	"messages.securitycraft:portable_tune_player.no_tune": "利用可能な音楽はありません。",
+	"messages.securitycraft:sc_help": "%s: %s + %s",
+	"messages.securitycraft:formattedCoordinates": "X: %1$s Y: %2$s Z: %3$s",
+
+	"jei.securitycraft.category.reinforcing": "ブロックの強化",
+	"jei.securitycraft.category.unreinforcing": "強化の除去",
+
+	"config.waila.plugin_securitycraft": "SecurityCraft",
```
[the remark this answers is not in the document]
Comment (momo-i) [reply src/main/resources/assets/securitycraft/lang/ja_jp.json]: waila.plugin_securitycraft line removed.

File: src/main/resources/assets/securitycraft/lang/ja_jp.json [hunk 5]
```diff
@@ -0,0 +1,1196 @@
...
+	"gui.securitycraft:sonic_security_system.stop_recording": "録音停止",
+	"gui.securitycraft:sonic_security_system.recording.clear": "録音クリア",
+	"gui.securitycraft:sonic_security_system.recording.notes": "録音済メモ: %s",
+	"gui.securitycraft:sonic_security_system.recording.play": "録音再生",
+	"gui.securitycraft:sonic_security_system.power.on": "パワー: オン",
+	"gui.securitycraft:sonic_security_system.power.off": "パワー: オフ",
+	"gui.securitycraft:sonic_security_system.sound": "サウンド:",
+	"gui.securitycraft:sonic_security_system.recording": "録音中...",
+	"gui.securitycraft:sonic_security_system.listening": "傾聴中...",
+	"gui.securitycraft:customize.tooltip": "%s (%s)",
```
[the remark this answers is not in the document]
Comment (momo-i) [reply src/main/resources/assets/securitycraft/lang/ja_jp.json]: customize.tooltip line removed.

**Review by bl4ckscor3 — APPROVED** [review]

**Pull request merged**